kpi/tests/test_cloning.py

```diff
@@ -16,6 +16,8 @@
 )
 from .kpi_test_case import KpiTestCase
 from .test_assets import AssetsTestCase
+from kpi.constants import *
```

Comment:
:grin: If importing [lots of constants](https://github.com/kobotoolbox/kpi/pull/1806/files#diff-7efb57e207546d622f02cddb15ffa899R99) is onerous, how about `from kpi import constants` and then using references like `constants.ASSET_TYPE_ARG_NAME` afterwards?

Reply:
I agree with you. I forgot to specify the variables after refactoring the code. I will import the needed constants. I don't want to use `constants.VARIABLE`